### ISSUE TITLE
Linux VST3: Camomile .so library files copied but not renamed in build

### ISSUE THREAD

**Issue by GModal** [pierreguillot/Camomile#192]
2. **Describe your environment**
Debian Buster, stable
Camomile 1.0.7 Beta14 (Linux 64 binary zip)
          
3. **Describe the problem or the feature request**

Linux script invoked by opening shell in Camomile dir, and entering "./camomile", to build all plugins.

When building the plugins, "CamomileFx.so" not renamed to "ProjName".so

The naming behavior isn't necessarily consistent, when the files are renamed in "build" they seem to retain that name. Probably because the library files don't change between builds.

This holds true for all the included plugin examples: AlmondOrgan, Bulgroz, etc. 

For those examples, the build script outputs (this probably exposes a separate minor issue, the version #). 
``` 
generating ttl AlmondOrgan in ./builds/AlmondOrgan.lv2/AlmondOrgan.so
error : patch has been created for a newer version of the plugin v1.0.7
Writing manifest.ttl... done!
Writing AlmondOrgan.ttl... done! 
```

**Comment by pierreguillot** [comment]
Can you try using this version? 
[CamomileLinux64.zip](https://github.com/pierreguillot/Camomile/files/5103112/CamomileLinux64.zip)
This should also fix https://github.com/pierreguillot/Camomile/issues/193.

**Comment by GModal** [comment]
Yes, this fixes both issues here. Tested your bundled examples, and my own patches.
Thanks!!
